### ISSUE TITLE
@Getter("model") ignored if getModel() exists and is declared after the actual getter

### ISSUE THREAD

**Issue by PtiPingouin** [gfx/Android-Orma#293]
Consider the following example:
```
protected static final String MEMBER_MODEL = "model";

@SerializedName(MEMBER_MODEL)
protected WrappedModel mWrappedModel;

public Model getModel() {
    return mWrappedModel.unwrap();
}

@Getter(MEMBER_MODEL)
public WrappedModel getWrappedModel() {
    return mWrappedModel;
}
```
It works fine.
However, if I now declare getModel() after getWrappedModel(), @Getter will be ignored, and getModel() will be used as a getter for mWrappedModel. That is, there will be a mismatch between the return types (and a build error).

**Comment by PtiPingouin** [comment]
Same goes for setters.

**Comment by gfx** [comment]
👀 

Looks a bug. Will fix it in a few days!

**Comment by gfx** [comment]
Fixed in #296 and released as v3.0.0-rc3. Thanks.
